tests/test_worst_case_attack.py

```diff
@@ -20,6 +21,57 @@ def clean_up(name):
         os.remove(name)
 
 
+def test_config_file_arguments_parsin():
```

Comment:
These tests don't have any assertions so aren't actually checking anything?

Also, the OS calls to execute the command line wont work on linux (including the CI runner) because it needs to call `python3` and not `python` - these really should have been updated in all the examples too but that's an issue beyond this PR.

Reply:
@rpreen can you make an issue to address this please?